extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -262,7 +262,7 @@ export abstract class AzureAuth implements vscode.Disposable {
 
 		const tokenClaims: TokenClaims = this.getTokenClaims(accessTokenString);
 
-		const userKey = tokenClaims.sub ?? tokenClaims.oid;
+		const userKey = tokenClaims.home_oid ?? tokenClaims.oid ?? tokenClaims.unique_name ?? tokenClaims.sub;
```

Comment:
Can you capture some of your learnings here? Just a basic rundown of these values and they're expected

Reply:
Yep!